packages/axes/src/inputType/PanInput.ts

```diff
@@ -353,6 +358,9 @@ export class PanInput implements InputType {
     this._observer = observer;
     this._enabled = true;
     this._activeEvent = activeEvent;
+    if (this.options.preventClickOnDrag) {
+      this.element?.addEventListener("click", this._preventClickWhenDragged);
```

Comment:
I think it shouldn't be activated if the element doesn't exist.
Like,

```js
if (!activeEvent || !this.element) {
  return;
}
```

Actually, I wonder in what case the element be `null`, as it receives the element in the constructor.
Please check whether the type is incorrect for this case.

Reply:
This seems to be unnecessary since existence of the element should be guaranteed by user at the timing of `_attachElementEvent`, unlike `_detachElementEvent` which element may not exist while input is disconnected. 
I think we should throw an error if the target element does not exist at `_attachElementEvent`.